apps/wistia-videos/src/components/Field.tsx

```diff
@@ -31,37 +33,47 @@ const StyledImageContainer = styled.div`
 
 const Field = (props: FieldProps) => {
   const { sdk } = props;
-  const [data, updateData] = useState<WistiaItem[] | []>([]);
+  const [data, updateData] = useState<WistiaVideo[] | []>([]);
```

Comment:
nit: You can get rid of the basic `[]` typing and just make it 
```
  const [data, updateData] = useState<WistiaVideo[]>([]);
```
and do the following below